### PR TITLE
[core modules] Centralize core account addresses.

### PR DESCRIPTION
**Pull request by tzakian**
This is the first step towards #4267. This just centralizes all of the core code addresses in one module. Note that the function naming conventions are intentionally violated here since they will eventually be replaced with constants once we have those.

### CODE REVIEW

File: language/stdlib/modules/Addresses.move [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    public fun TRANSACTION_FEE_ADDRESS(): address {
+        0xFEE
+    }
+
+    public fun DEFAULT_CONFIG_ADDRESS(): address {
+        0xF1A95
+    }
+}
+}
```
Comment (sblackshear) [inline language/stdlib/modules/Addresses.move:29]: <3

File: language/stdlib/modules/Addresses.move [hunk 2]
```diff
@@ -0,0 +1,29 @@
+address 0x0 {
+module CoreAddresses {
+    public fun ASSOCIATION_ROOT_ADDRESS(): address {
```
Comment (sblackshear) [inline language/stdlib/modules/Addresses.move:3]: Comments for each of these might be useful

File: language/stdlib/modules/Addresses.move [hunk 3]
```diff
@@ -0,0 +1,29 @@
+address 0x0 {
+module CoreAddresses {
```
Comment (sblackshear) [inline language/stdlib/modules/Addresses.move:2]: Nit: rename either the file or the module?

Comment (tzakian) [reply language/stdlib/modules/Addresses.move:2]: Woops...forgot to rename the file after renaming the module....good spot!

**Pull request closed without merging**